packages/service/pyproject.toml

```
@@ -7,7 +7,7 @@ extend_exclude = '.tox/,*_pb2_grpc.py,*_pb2_grpc.pyi,*_pb2.py,*_pb2.pyi,ni_measu
 
 [tool.poetry]
 name = "ni_measurement_plugin_sdk_service"
-version = "2.1.0-dev0"
+version = "2.1.0-dev1"
```

The "2.1.0-dev1" is already pre-released. This should be `dev2` now.
```suggestion
version = "2.1.0-dev2"
```